v2/src/components/appInfoForm/index.tsx

```diff
@@ -724,4 +785,26 @@ export default class AppInfoForm extends React.PureComponent<PropsWithChildren<P
 
         return Object.keys(validationErrors).length === 0;
     }
+
+    scrollToElement() {
+        if (Boolean(this.containerRef.current)) {
+            window.scrollTo({
+                top: window.scrollY + (this.containerRef.current?.parentElement?.getBoundingClientRect().top ?? 0),
+                behavior: 'smooth'
+            });
+        }
+    }
+
+    private onReceiveAttribute(mutations: MutationRecord[]) {
+        if (isReceivingFirstFormAttr(mutations)) {
+            this.setState({ firstAppInfoForm: this.isFirstVisibleAppInfoForm() })
+        }
```

Comment:
the `if (isReceivingFirstFormAttr(mutations))` is the fix for case of the following structure
```tsx
<details>
  <summary>There is form inside</summary>  
  <AppInfoForm askForAPIDomain>Content of 1st AppInfoForm inside detail</AppInfoForm>
</details>

<AppInfoForm askForAPIDomain>Content of 2nd AppInfoForm on outer</AppInfoForm>
```